google/cloud/forseti/notifier/notifiers/gcs_inv_summary.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Upload inventory summary to GCS."""
+
+from google.cloud.forseti.common.gcp_api import storage
+from google.cloud.forseti.common.util import date_time
+from google.cloud.forseti.common.util import logger
+from google.cloud.forseti.common.util import string_formats
+from google.cloud.forseti.notifier.notifiers.base_notification import (
+    BaseNotification)
+
+
+LOGGER = logger.get_logger(__name__)
+
+
+class GcsInvSummary(object):
```

Comment:
I lean toward not having this to be `GCS` specific, and rather call it `InventorySummary`  This way, it can be extended to other notifications like as slack or email.

Reply:
Done.